web/src/recent_topics_ui.js

```diff
@@ -859,6 +859,7 @@ export function complete_rerender() {
         sort_fields: {
             stream_sort,
             topic_sort,
+            ...ListWidget.generic_sort_functions("numeric", ["last_msg_id"]),
```

Comment:
How can I verify that the diffs in this commit don't change the sorting configuration?

Reply:
I believe for that you will require some manual work 🥲, one thing I already did to verify that the sorting configuration remains the same was to check the corresponding templates which have these table headers which sets the `data-sort` prop and manually see which of these are using generic props.

Reply:
OK, I'll trust you to have checked it carefully; it certainly seems plausible that this configuration is the same as before.